Create ui-c8y-1023-97-7-fix-activity-log-blank-gap-during-load-and-date-pipe-ng.md (#4882)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1023-97-7-fix-activity-log-blank-gap-during-load-and-date-pipe-ng.md

---
date: ""

> title: Activity log displays correctly without blank gaps or console errors

Reviewer: ```suggestion
title: Activity log displays dates correctly and loads without visual gaps or console errors
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67114
version: 1023.97.7
---

> The Activity log in applications previously displayed a blank gap while loading archive entries and dates for these entries were not displayed (also the console was flooded with DatePipe NG02100 errors). The Activity log now loads archived entries smoothly without blank gaps, date for each entry is displayed and the DatePipe errors no longer appear in the console.

Reviewer: ```suggestion
The Activity log in applications previously displayed a blank gap while loading archive entries and dates for these entries were not displayed (also the console was flooded with DatePipe NG02100 errors). The Activity log now loads archive entries smoothly without blank gaps, date for each entry is displayed and the DatePipe errors no longer appear in the console.
```

> The Activity log in applications previously displayed a blank gap while loading archived entries, and the console was flooded with DatePipe NG02100 errors that did not affect functionality but created noise in the developer console. The Activity log now loads archived entries smoothly without blank gaps, and the DatePipe errors no longer appear in the console. This improves the user experience by providing a cleaner loading state and eliminates console errors that could mask other important issues during development and troubleshooting.

Reviewer: "archived entires" - "archives" or "archive entries"? these are archive files, but they are not archived, one of them is currently active and deployed
"did not affect functionality" - wasn't it preventing the date from being displayed? ("The date is dropped.")

> The Activity log in applications previously displayed a blank gap while loading archived entries, and the console was flooded with DatePipe NG02100 errors that did not affect functionality but created noise in the developer console. The Activity log now loads archived entries smoothly without blank gaps, and the DatePipe errors no longer appear in the console. This improves the user experience by providing a cleaner loading state and eliminates console errors that could mask other important issues during development and troubleshooting.

Reviewer 2: of course, it should be "archive entries".
```suggestion
The Activity log in applications previously displayed a blank gap while loading archive entries and dates for these entries were not displayed (also the console was flooded with DatePipe NG02100 errors). The Activity log now loads archived entries smoothly without blank gaps, date for each entry is displayed and the DatePipe errors no longer appear in the console.
```
